### PR TITLE
TMA-735 Failed To Consume Event Message to S3

### PR DESCRIPTION
**Pull request by abdulkalam-govuk**
Code clean up done as JsonPath made it simpler.

### CODE REVIEW

**Review by zaydgds — CHANGES_REQUESTED** [review]

File: event-processing/lambda/models/redacted-event.ts [hunk 1]
```diff
@@ -47,68 +48,67 @@ export class RedactedAuditEvent  implements  IRedactedAuditEvent {
...
+            json: snsMessage
+        })[0];
+
+        redactedAuditData.reIngestCount = JSONPath({
+            path: "$.reIngestCount",
+            json: snsMessage
+        })[0];
+
+
+        if (JSONPath({
```
[the remark this answers is not in the document]
Comment (abdulkalam-govuk) [reply event-processing/lambda/models/redacted-event.ts]: DONE

File: event-processing/lambda/models/redacted-event.ts [hunk 2]
```diff
@@ -47,68 +48,67 @@ export class RedactedAuditEvent  implements  IRedactedAuditEvent {
...
+            })[0];
+        }
 
-            if ((user as IRedactedAuditEventUserMessage).govuk_signin_journey_id && (user as IRedactedAuditEventUserMessage).user_id) {
-                this.user = {
-                    govuk_signin_journey_id: (user as IRedactedAuditEventUserMessage).govuk_signin_journey_id,
-                    user_id: (user as IRedactedAuditEventUserMessage).user_id,
-                } as IRedactedAuditEventUserMessage;
-            }
+        if (JSONPath({
```
[the remark this answers is not in the document]
Comment (abdulkalam-govuk) [reply event-processing/lambda/models/redacted-event.ts]: DONE

File: event-processing/lambda/models/redacted-event.ts [hunk 3]
```diff
@@ -47,68 +48,67 @@ export class RedactedAuditEvent  implements  IRedactedAuditEvent {
...
-    };
-}
+    function createBaseRedactedAuditEventUserMessage(): IRedactedAuditEventUserMessage {
+        return {
+            user_id: '',
+            govuk_signin_journey_id: '',
+        };
+    }
 
 export class RedactedAuditEventUserMessage {
```
[the remark this answers is not in the document]
Comment (abdulkalam-govuk) [reply event-processing/lambda/models/redacted-event.ts]: Remove the unused

File: event-processing/lambda/services/redacted-service.ts [hunk 4]
```diff
@@ -1,7 +1,4 @@
-import {EnrichedAuditEvent, IEnrichedAuditEvent} from '../models/enriched-audit-event';
-import {IRedactedAuditEvent, RedactedAuditEvent} from '../models/redacted-event';
-import {AuditEvent, IAuditEvent} from "../models/audit-event";
-import {CleansedEvent, ICleansedEvent} from "../models/cleansed-event";
+import {IRedactedAuditEvent,  RedactedAuditEvent} from '../models/redacted-event';
 
 export class RedactedService {
```
[the remark this answers is not in the document]
Comment (abdulkalam-govuk) [reply event-processing/lambda/services/redacted-service.ts]: Rename DONE

**Review by zaydgds — APPROVED** [review]

**Pull request merged**